pandas/core/internals/blocks.py

```diff
@@ -386,7 +386,7 @@ def apply(self, func, **kwargs):
 
         return result
 
-    def fillna(self, value, limit=None, inplace=False, downcast=None):
+    def fillna(self, value, axis=0, limit=None, inplace=False, downcast=None):
```

Comment:
As is, this is a breaking API change.  To be safe, you would need to put it at the end.

Reply:
Can you also add an annotation for `axis`? Should be importable from `pandas._typing`

Reply:
> As is, this is a breaking API change. To be safe, you would need to put `axis` at the end.

I have just done this, thanks.

Reply:
> Can you also add an annotation for `axis`? Should be importable from `pandas._typing`

Sorry, I'm not sure what that means.  Could you direct me to an example or docs?

Reply:
Yea so a variable annotation. You'd essentially do:

```
from pandas._typing import Axis

def fillna(..., axis: Axis = 0):
```

Reply:
OK, thanks.  I have pushed this.